src/hook/pauseResume.js

```diff
@@ -0,0 +1,318 @@
+/**
+* Copyright (c) 2015 NAVER Corp.
+* egjs projects are licensed under the MIT license
+*/
+eg.module("pauseResume", ["jQuery"], function($) {
+	"use strict";
+
+	var animateFn = $.fn.animate;
+	var stopFn = $.fn.stop;
+	var uuid = 1;
+	var getStyles;
+
+	if (window.getComputedStyle) {
```

Comment:
@mixed Thank you for your kindness. And I applied your reviews. As you tell me, I tested that While a DOM element is not changing, the `getComputedStyle` makes tiny performance issue which we don't have to worry. I think Changing from caching the getComputed value as a `computed` variable to $.css will be OK.

But I want to emphasize BIG differences on this commit.

I changed the time to calculate absolute value from relative value and the way to get previous value.

**BEFORE**
1. calculation time: on animate call time
2. way to get previous value: cached value each calculated on animate call time.


**AFTER**
1. calculation time: on animate complete time
2. way to get previous value: call $.css when current value is relative.

Problems that **'BEFORE'** had is below.

CSS value can be changed between before animation is done like below.

```js
this.$el
  .animate({"left": "100px"}, duration, function() {
  /**
   * This makes an changes.
   */
    $.style(this, "left", 0);
  })
  .animate({"left": "+=100px"});//MUST move left value based on 0.
```

Code above does not act on **BEFORE** pause/resume code. because Next animation refers to the value that is not applied custom changes(like on complete, step, progress)